src/python/arcor2_arserver/lock.py

```diff
@@ -1,40 +1,40 @@
 import asyncio
 from datetime import datetime
-from typing import Optional, Dict, AsyncGenerator
+from typing import Optional, Dict, AsyncGenerator, List, Union
 
 from arcor2.cached import UpdateableCachedProject, UpdateableCachedScene
 from arcor2.exceptions import CannotUnlock
 
 
 class Lock:
 
-    SERVER_NAME = "SERVER"
-    SCENE_NAME = "SCENE"
-    PROJECT_NAME = "PROJECT"
+    SERVER_NAME: str = "SERVER"
+    SCENE_NAME: str = "SCENE"
+    PROJECT_NAME: str = "PROJECT"
 
-    CLEAN_DELAY = 10
-    LOCK_TIMEOUT = 300  # 5 minutes
+    CLEAN_DELAY: int = 10
+    LOCK_TIMEOUT: int = 300  # 5 minutes
 
     class Data:
         __slots__ = "owners", "timestamp", "count"
 
-        def __init__(self, owner):
-            self.owners = [owner]
-            self.timestamp = datetime.utcnow()
-            self.count = 1
+        def __init__(self, owner) -> None:
```

Comment:
Ještě tady chybí `:str` :) Zkoušel jste pustit `./pants typecheck ::`? Myslím že by vás to mělo na takové případy upozornit.